tests/test_globalconstraints.py

```diff
@@ -470,6 +470,15 @@ def test_bounds_maximum(self):
         self.assertFalse(cp.Model(expr<lb).solve())
         self.assertFalse(cp.Model(expr>ub).solve())
 
+    def test_bounds_abs(self):
```

Comment:
Should we also put normal tests for abs besides testing the bounds? I mean, asserting that the value is the expected one, as we do with the rest of the global constraints/functions